src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js

```diff
@@ -23,20 +23,27 @@ export const ContentContainer = styled.div`
 
 export const ListContainer = styled.div`
   display: flex;
-  flex-flow: wrap;
+  flex-wrap: wrap;
   & > div:nth-child(odd) {
     background: ${colors.smoothGrey};
   }
   & > div:nth-child(n + 4) {
-    margin-top: ${spacings.spacesLvl2};
+    margin-top: ${spacings.spacesLvl0};
+    @media (min-width: 768px) {
+      margin-top: ${spacings.spacesLvl2};
```

Comment:
Why do you need this here? I thought with just the grid it should adapt right?

Reply:
That margin is the intermediate and necessary space to read the data fluently, according to what Santi told me.
But in mobile it does not work and can be removed.
The grid adapts the boxes is true, but there are certain things that we will have to touch them in the elements.